packages/suite/src/components/suite/TransactionsGraph/components/CustomTooltipBase/index.tsx

```diff
@@ -120,13 +120,15 @@ const CustomTooltipBase = (props: Props) => {
                 ? 'month'
                 : 'day';
 
-        if (props.onShow && props.extendedDataForInterval) {
-            props.onShow(
-                props.extendedDataForInterval.findIndex(
-                    item => item.time === props.payload![0].payload.time,
-                ),
-            );
-        }
+        useEffect(() => {
+            if (props.onShow && props.extendedDataForInterval) {
+                props.onShow(
+                    props.extendedDataForInterval.findIndex(
+                        item => item.time === props.payload![0].payload.time,
+                    ),
+                );
+            }
+        }, [])
```

Comment:
Not sure about this one, but shouldn't `extendedDataForInterval` be in the dependencies?

Reply:
I'll check

Reply:
The thing is, I pass setState to a child component and it clashes with rendering of itself. I googled for a solution and found this https://stackoverflow.com/a/63289170/872981 - I applied useEffect and it works correctly now.

However you were right about the props, I added those and TS works OK now. 👍